src/main/java/com/knightexperience/app/gateway/privatedb/repository/CharacterRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.knightexperience.app.gateway.privatedb.repository;
+
+import com.knightexperience.app.gateway.privatedb.representation.Character;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface CharacterRepository extends JpaRepository<Character, Long> {
+
+    Optional<Character> findById(String characterId);
+
+    void deleteById(String characterId);
+}
```

Comment:
I think we don't need to write these methods here, instead we can just change JpaRepository parameters to <Character, String>

Reply:
Thanks, I'll change it. Really just copied from the todoBlock. It's better to change it everywhere before it start spreading everywhere.

Reply:
Sure!